### DISCUSSION TITLE
Zeek logs not appearing in Kibana - SO 2.3.10 Import

### DISCUSSION THREAD

**Discussion by DefxNull** [Security-Onion-Solutions/securityonion#5020] [Unsupported Versions] [answered]
BLUF: I run "sudo so-import-pcap ~/Downloads/***.pcap" but I cannot see the Zeek logs in Kibana.

Specs:
- ESXi 6.5.0
- SecurityOnion 2 2.3.10
- Install Type: Import

I can see the the import is successful and analyzes the PCAP with Zeek.  I also checked the directory "/nsm/import/eac***/logs/" and can see the log files as well as the content inside.  I am able to view the suricata logs in the Kibana Dashboards, but cannot find the Zeek logs.   I've restarted services and docker (sudo so-docker-refresh), rebooted SO (sudo reboot), as well as deleted the PCAP and reimported it, but I still cannot get the Zeek logs to be viewable in Kibana.  Anything I am missing here?  Thanks for the help.

![SO import output](https://user-images.githubusercontent.com/88286422/127783133-c626032c-3d87-4ef7-b6fd-63f8eb0b4692.jpg)
![Zeek logs](https://user-images.githubusercontent.com/88286422/127783274-b05ffb05-0fc3-4f08-8c91-3a190bf7e4fd.jpg)
![SO Kibana no zeek logs](https://user-images.githubusercontent.com/88286422/127783307-b1261f8e-a4d7-4a54-bb3a-320d4f28cce2.jpg)
![SO Kibana only suricata logs](https://user-images.githubusercontent.com/88286422/127783366-268444c1-8d0b-45b2-ab7c-044122af6f22.jpg)

**Comment by TOoSmOotH** [comment, marked as answer]
You are using a very old version of SO2. I would upgrade or re-install and try again.

**Reply by DefxNull** [reply]
Thanks Mike.  Both reinstalling the old .iso and the latest release worked.
